core/lib/authorization/capabilities.db.test.ts

```diff
@@ -0,0 +1,113 @@
+import { describe, expect, test } from "vitest";
+
+import { CoreSchemaType, MemberRole } from "db/public";
+import { Capabilities } from "db/src/public/Capabilities";
+import { MembershipType } from "db/src/public/MembershipType";
+
+import { seedCommunity } from "~/prisma/seed/seedCommunity";
+import { mockServerCode } from "../__tests__/utils";
+
+const { createForEachMockedTransaction } = await mockServerCode();
```

Comment:
This test currently fails because this doesn't seem to be mocking React.cache, though I have no idea why.
<img width="716" alt="image" src="https://github.com/user-attachments/assets/be824635-386d-468c-8367-9ef2b9fe4db6">

Reply:
The test passes if I put it in the pub.db.test.ts file, so I think I'm missing some setup step here but I have no idea which

Reply:
turns out this was because I had imported `mockServerCode` after `seedCommunity` and it has to come first! Probably this means we should be dynamically importing `seedCommunity` in these tests, but for now I just switched the import style from relative to using our ~/lib alias, which makes the mockServerCode import sort first.